_maps/map_files/Omegastation/job_changes.dm

```diff
@@ -0,0 +1,177 @@
+#define JOB_MODIFICATION_MAP_NAME "OmegaStation"
+
+/datum/job/New()
```

Comment:
In my own personal tests, the job setup hadn't changed at all. I joined in as CE roundstart and ended up in the Detective's bedroom.

This *may* be the fault of the actual job change routine, rather than what you're doing here, but regardless I would like if there were some fallback spawner landmarks for all roles this map isn't supposed to have.